flink-ml-python/pyflink/ml/lib/param.py

```diff
@@ -155,6 +155,27 @@ def input_cols(self) -> Tuple[str, ...]:
         return self.get_input_cols()
 
 
+class HasIndices(WithParams, ABC):
```

Comment:
Maybe we should implement it inside `vectorslicer.py`, instead of treating it as a common parameter. You can refer to the `K` parameter in `kmeans.py`.